accounts-db/src/accounts_db.rs

```diff
@@ -15776,13 +15773,13 @@ pub mod tests {
         // empty store
         let storage = accounts.create_and_insert_store(0, 1, "test");
         let storage_info = StorageSizeAndCountMap::default();
-        let accounts_map = accounts.process_storage_slot(&storage);
         accounts.generate_index_for_slot(
-            accounts_map,
+            &storage,
             0,
             0,
             &RentCollector::default(),
             &storage_info,
+            storage.approx_stored_count(),
```

Comment:
nit: maybe don't pass `approx_stored_count` in `generate_index_for_slot`, get it from `&storage` instead inside the fn?

Reply:
done